### ISSUE TITLE
Update puppeteer library

### ISSUE THREAD

**Issue by miguel-101** [screener-io/screener-storybook#153]
Hello,

Can you please update `puppeteer` library to the latest or at least a recent version? ([latest](https://github.com/puppeteer/puppeteer/releases/tag/v19.2.2) is **v19.2.2**)

The current one
```JSON
"puppeteer": "~1.17.0"
```
is very outdated and [maps to](https://github.com/puppeteer/puppeteer/blob/30e5b1a58edb8b1d94acdff00d64c76e76cf02a3/website/versioned_docs/version-19.2.2/chromium-support.md) an old `Chromium` version:

<img width="337" alt="image" src="https://user-images.githubusercontent.com/117231739/202602480-56957565-c285-45e9-a11c-8a8fcfa37f5f.png">

Having an old `Chromium` version makes our project fail in this line:
https://github.com/screener-io/screener-storybook/blob/a2ebbe8810780ea588ed7ceb93007a8865e7e2de/src/storybook.js#L42

due to `Chromium` not loading `.../iframe.html` file because of the old js engine not recognizing new syntaxis and throwing errors
![image](https://user-images.githubusercontent.com/117231739/202605972-38223c7e-b022-4b6f-8e96-59dacd7deaf0.png)
![image](https://user-images.githubusercontent.com/117231739/202606123-2d8e9706-6f2e-4bff-a7df-e56064e669ac.png)

**Comment by dswiz77** [comment]
Hi Miguel, I just wanted to confirm we got your request and will look into this but won't be able to give you an ETA on when it will get resolved.

**Comment by sauce-tartar** [comment]
@miguel-101 hello, is there a minimum version of Chrome you have in mind?

We have a series of updates in mind for screener-storybook, leading those and active is the storyStoreV7 feature but could include a Puppeteer upgrade there.  Under experiment it looks like we can jump right to 19 but may choose a smaller step as an interim release.

_p.s. I also found it entertaining that the current version is 18 major versions ahead of us_ 😆😢

**Comment by sauce-tartar** [comment]
@miguel-101 which version of Storybook are you using, and are you willing to share the contents of your `.storybook/main.js` or the features in use?

I'm wondering now if this is caused by the Story Store v7 architectural changes.

`window.__screener_storybook__ is not a function` could be caused by that, in Story Store V7 the framework hook being used is going away.  We're testing it's replacement right now.

**Comment by miguel-101** [comment]
Hey @sauce-tartar thanks for taking a look at this issue!

Let me share a bit more context:

The syntaxis that broke out storybook build was `Nullish coalescing operator (??)`.
![image](https://user-images.githubusercontent.com/117231739/204592583-5e2300f1-cf7c-49b3-a6bc-ca51da1c5d54.png)

I found it by launching screener/chromium while debugging in headless mode off ☝️
https://github.com/screener-io/screener-storybook/blob/a2ebbe8810780ea588ed7ceb93007a8865e7e2de/src/storybook.js#L91 

Changed ☝️ to 👇

```JS
var launchOptions = {headless: false};
```

Since the build contained errors, `Storybook` didn't add the `__screener_storybook__` fn to the `window` obj. So, it threw the `not a function` error.
![image](https://user-images.githubusercontent.com/117231739/204596412-49113f5f-033a-434d-9b34-c7bcb0c5c4f5.png)

### Questions:

> is there a minimum version of Chrome you have in mind?

That `operator (??)` seems to be [compatible with Chromium version 80+
](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#browser_compatibility) ([Puppeteer v2.1.0](https://github.com/puppeteer/puppeteer/blob/v2.1.0/docs/api.md))

> which version of Storybook are you using
```JSON
"@storybook/react": "6.3.8"
```

> and are you willing to share the contents of your .storybook/main.js or the features in use?

About `main.js`, we noticed this started happening when we tried to use `webpack5`
``` JS
  core: {
    builder: 'webpack5',
  }
```
Our issue got fixed temporarily by not using this `webpack` config ☝️

**Comment by sauce-tartar** [comment]
Hi @miguel-101, I'm happy to share we have an upgrade to Puppeteer 18 in our new Alpha, I wonder if you might try it to see if it works for you.  Additionally the `window.__screener_storybook__` injected global hook approach is removed.

We have more about the Puppeteer version in the README on the PR here:

https://github.com/screener-io/screener-storybook/pull/157

```
screener-storybook@alpha
```

### webpack5

This alpha does *not* address webpack5, but:
* we know this is a must for Storybook 7
* perhaps the change in our approach will work for you as well, meaning as we no longer rely on injecting our own hook and perhaps this change alone would find your `6.3.8` + webpack 5 working

Lastly thanks for sharing your config, I'll bring this up with the team and adding more 6.3 into our CI matrix seems like a good idea.

**Comment by sauce-tartar** [comment]
@miguel-101, we've now released Screener Storybook 1.0 with Puppeteer 18.  Additionally the internal error you were receiving related to `__screener_storybook__` should not happen as we have refactored away from the approach that caused this.

Okay to close the issue?

**Comment by miguel-101** [comment]
Thank you @sauce-tartar, I've tested this new version released (v1.0) and so far it's working great!

We'll update the lib on our end and let you know of any issues. I'm closing this issue
